selfdrive/loggerd/uploader.py

```diff
@@ -236,14 +237,19 @@ def uploader_fn(exit_event):
   uploader = Uploader(dongle_id, ROOT)
 
   backoff = 0.1
+  counter = 0
+  should_upload = False
   while not exit_event.is_set():
     offroad = params.get("IsOffroad") == b'1'
     allow_raw_upload = (params.get("IsUploadRawEnabled") != b"0") and offroad
-    on_hotspot = is_on_hotspot()
-    on_wifi = is_on_wifi()
-    should_upload = on_wifi and not on_hotspot
+    check_network = (counter % 12 == 0 if offroad else True)
```

Comment:
If I am on cellular and only have raw files to upload, then I get on wifi it can take 12 minutes now for it to start uploading raw files (when offroad)?  Seems like this counter shouldn't be necessary when sleeping for 60 seconds?

Reply:
@gregjhogan Yeah, I noticed this too, I thought it was being a bit weird until I saw this PR. Is this intentional?

Reply:
I think this part of the code is updated on master, so it shouldn't be an issue.

Reply:
@grekiki Ah, has it yet to hit a release? I'll go take a look then.